samples/word/99-preview-apis/manage-styles.yaml

```diff
@@ -0,0 +1,357 @@
+                <input id="new-style-name"/>
+            </p>
+                <p />
+                <label style="margin-left: 20px">Type:</label>
+                <select id="new-style-type">
+                <option>Paragraph</option>
+            <option>Character</option>  
+            <option>List</option>  
+            <option>Table</option>
+          </select>
+                <p />
+                <button id="add-style" class="ms-Button">
+            <span class="ms-Button-label">Add style</span>
+          </button>
+
+                <h4>Use custom style</h4>
+                <label style="margin-left: 20px">Style name:</label>
+                <input id="style-name-to-use"/>
+                <p />
+                <button id="properties" class="ms-Button">
+              <span class="ms-Button-label">Get properties</span>
+            </button>
+                <p />
+                <button id="apply-style" class="ms-Button">
+              <span class="ms-Button-label">Apply style to paragraph</span>
+            </button>
+
+                <h4>Update custom style</h4>
+                <label style="margin-left: 20px">Style name:</label>
+                <input id="style-name"/>
+                <p />
+                <div>Set font color to red and font size to 20.</div>
+                <button id="font-properties" class="ms-Button">
+            <span class="ms-Button-label">Set font properties</span>
+          </button>
+                <p />
+                <div>Set paragraph left indent size to 30 and alignment to center.</div>
+                <button id="paragraph-format" class="ms-Button">
+            <span class="ms-Button-label">Set paragraph format</span>
+          </button>
+                <p />
+                <div>Set outside border type to dashed, width to pt. 0.25, and color to green.</div>
+                <button id="border-properties" class="ms-Button">
+                    <span class="ms-Button-label">Set border properties</span>
+                  </button>
+
+                <h4>Delete custom style</h4>
+                <p>
+                    <label style="margin-left: 20px">Style name:</label>
+                    <input id="style-name-to-delete"/>
+                    <button id="delete-style" class="ms-Button">
+              <span class="ms-Button-label">Delete style</span>
+              </button>
+                </p>
+                <h4>Get style info from external document</h4>
+                <p>Select a Word document to get its style info.</p>
+                <form>
+                    <input type="file" id="file" />
+                </form>
+                    <br>
+                    <p>Get style info from the selected document.</p>
+                    <button id="get-external-styles" class="ms-Button">
+                      <span class="ms-Button-label">Get styles</span>
+                    </button>
+        </section>
```

Comment:
Two things here:

1. Consider making the external file stuff a separate sample. It seems distinct enough.
2. The indentation in the HTML seems off.